packages/teleterm/src/services/grpcCredentials/makeCert/makeCert.ts

```diff
@@ -69,6 +72,15 @@ export async function makeCert({
       digitalSignature: true,
       keyEncipherment: true,
     },
+    {
+      name: 'subjectAltName',
+      altNames: [
+        {
+          type: 2, // DNS type
+          value: commonName,
+        },
+      ],
+    },
```

Comment:
To my surprise, Go didn't complain when a cert without this was used as a client cert, but it was a problem when I tried to use it as a server cert for the client credentials.